server/public/model/system.go

```diff
@@ -119,6 +119,9 @@ type SupportPacket struct {
 	LicenseTo             string `yaml:"license_to"`
 	LicenseSupportedUsers int    `yaml:"license_supported_users,omitempty"`
 	LicenseIsTrial        string `yaml:"license_is_trial,omitempty"`
+	CustomerID            string `yaml:"customer_id"`
+	LicenseID             string `yaml:"license_id"`
+	DiagnosticID          string `yaml:"diagnostic_id"`
```

Comment:
What is the legal impact of including the anonymized telemetry ID in the Support Packet? Mattermost is now able to identify usage patterns for a specific customer using webapp telemetry.

Reply:
Interesting point, I asked for feedback. Thanks! https://hub.mattermost.com/private-core/pl/u89jqbyfqjy4dx85tujiqdzapc